### PR TITLE
feature/remove-inline-accounting

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========
Remove inline_accounting table.

NEWS file entries
=======================
Enhancements
-------------
* Remove unused table inline_accounting

Delete branch after merge
=========================
NO

Checklist
=========
- [ ] Document the feature
- [x] Add unit tests
- [ ] Add acceptance tests (TestLink)

### CODE REVIEW

**Comment by extrafu** [comment]
@jrouzierinverse rebase  and @nqb / @julsemaan review and merge if fine

**Review by nqb — CHANGES_REQUESTED** [review]
### Missing files

There is some references to `inline.accounting` inside `pf.conf.defaults` and `documentation.conf`

### Drop of DB table

I see you dropped `inline_accounting` table.
Tell me if I'm wrong but it means that we lost "bandwidth history" for nodes.
I'm not able to see if we lost an old history.

File: db/upgrade-X.X.X-X.Y.Z.sql [hunk 1]
```diff
@@ -50,6 +50,9 @@ DELIMITER ;
 call ValidateVersion;
 DROP PROCEDURE IF EXISTS ValidateVersion;
 
+\! echo "Dropping table inline_accounting"
+DROP TABLE `inline_accounting`;
```
Comment (nqb) [inline db/upgrade-X.X.X-X.Y.Z.sql]:
```suggestion
DROP TABLE IF EXISTS `inline_accounting`;
```

**Review by nqb — CHANGES_REQUESTED** [review]

**Comment by julsemaan** [comment]
Other than @nqb's comments, it looks fine to me

**Comment by extrafu** [comment]
@jrouzierinverse Finish the work on that PR before tackling bugs.

**Comment by nqb** [comment]
@jrouzierinverse: there is still some references to `inline.accounting` inside `pf.conf.defaults` and `documentation.conf`

**Comment by nqb** [comment]
@jrouzierinverse, I did the removal for old config settings. Let me know if it's ok.

I just need an answer on "Drop of DB table" topic I mentioned and we should be ready to merge.

**Comment by nqb** [comment]
Bump @jrouzierinverse

**Comment by nqb** [comment]
Task `inline_accounting_maintenance` run each minute and mark inactive sessions based on:
- `$accounting_session_timeout` (default value: 300 seconds)
- one day history

Then task update `bandwidth_balance` column value  in `node` table.

There is also some logical related to management of security event.

I didn't see any jobs that cleanup `inline_accounting` table so it means all data will be lost. It should impact only users that extract data from this table for external usage. I will merge this PR as-is and add a note in Upgrade guide.

**Pull request merged**